### DISCUSSION TITLE
How to install NocoDB with default MySQL [Sep 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3748] [Discord Threads]
> 🙋‍♂️ **nilesh1991**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### You can define `NC_DB` environment variable  
#  
> 👨‍💻 **pranavxc**  
#### Sample value - `mysql2://localhost:3306?u=root&p=password&d=nocodb`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016958217558949959)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
